site/static/css/styles.css

```diff
@@ -3276,6 +3276,16 @@ h6:first-child {
     font-size: .9rem;
 }
 
+.blog-pagination a {
+    text-decoration: none;
```

Comment:
Is this really needed?

Reply:
Not necessary. I've removed these styles.